src/infrastructure/dataSchemas/allDataSchema.ts

```diff
@@ -0,0 +1,10 @@
+import { SemanticVersion } from '@/@types/types'
+import { configDataSchema } from './configDataSchema'
+import { PlotDataSchema } from './plotDataSchema'
+
+export interface allDataSchema {
+  appVersion: SemanticVersion
```

Comment:
As a type, it's a semantic version, and when actually used, it's the app version. This usage looks good.